apps/ledger-live-mobile/src/newArch/features/Accounts/screens/AccountsList/index.tsx

```diff
@@ -142,13 +60,21 @@ export default function AccountsList({ route }: Props) {
               onClick={onClick}
             />
           )}
-          <AccounstListView
-            sourceScreenName={sourceScreenName}
-            isSyncEnabled={isSyncEnabled}
-            specificAccounts={specificAccounts}
-          />
+          {!hasNoAccount ? (
```

Comment:
Can we avoid doing negative conditions when possible ? I find it more readable to put it the other way : `hasNoAccount ? <EmptyList /> : <List />`
Wdyt ?